examples/cookbook/search_with_sources.py

```diff
@@ -84,10 +86,10 @@ class SearchResponse(BaseModel):
     {question}
     """
 )
-def extract(question: str, results: str): ...
+def extract(question: str, results: str) -> None: ...
 
 
-def run(question: str):
+def run(question: str) -> Any:
```

Comment:
any seems too broad, if we can't type this better maybe we need to update the example?